examples/with_pyo3/src/lib.rs

```diff
@@ -9,18 +9,17 @@ pub fn add(left: usize, right: usize) -> usize {
 #[pyfunction]
 pub fn arange(n: usize) -> ManagerCtx<Vec<f32>> {
     let v: Vec<f32> = (0..n).map(|x| x as f32).collect();
-    let tensor = ManagerCtx::new(v);
-    tensor
+    ManagerCtx::new(v)
 }
 
 #[pyfunction]
-pub fn tensordict(py: Python<'_>) -> PyResult<&PyDict> {
-    let dic = PyDict::new(py);
+pub fn tensordict(py: Python<'_>) -> PyResult<Py<PyDict>> {
```

Comment:
Maybe we should return `Bound<'_, PyDict>` here. And don't have to `unbind`.

Reply:
Ah, yes. You're correct.